exercises/dominoes/dominoes_test.rb

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env ruby
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'dominoes'
+
+# Test data version:
+# 82eb00d
+class DominoesTest < Minitest::Test
+  def test_empty_input_empty_output
+    # skip
+    input_dominoes = []
+    output_chain = Dominoes.chain(input_dominoes)
+    assert_correct_chain(input_dominoes, output_chain)
+  end
+
+  def test_singleton_input_singleton_output
+    skip
+    input_dominoes = [[1, 1]]
+    output_chain = Dominoes.chain(input_dominoes)
+    assert_correct_chain(input_dominoes, output_chain)
+  end
+
+  def test_singleton_that_can_not_be_chained
+    skip
+    input_dominoes = [[1, 2]]
+    output_chain = Dominoes.chain(input_dominoes)
+    assert_nil output_chain, "There should be no chain for #{input_dominoes}"
```

Comment:
From the point of view of the student looking at this test generator: There is a lot of repetitive `"There should be no chain for #{input_dominoes}"`. Moving it into `refute_correct_chain` removes this repetition. So that's one point in favour of it. One point against is that it'd be very simple, but that's very minor.

Here is another proposal that might also go along with `refute_correct_chain`:

Should `output_chain = Dominoes.chain(input_dominoes)` go inside `assert_correct_chain` and `refute_correct_chain`? Again it would reduce duplication, and then those two methods would only have to take in the `input_dominoes`. Though this might make it harder for students to understand what they're supposed to implement  (they have to look for the definitions of `{assert,refute}_correct_chain`).

So currently: I am thinking yes to `refute_correct_chain` and no to "move `output_chain = Dominoes.chain(input_dominoes) into `{assert,refute}_correct_chain`"